tests/stavenote_tests.ts

```diff
@@ -845,7 +845,7 @@ function drawBeamStyles(options: TestOptions, contextBuilder: ContextBuilder): v
   staveNotes[1].setKeyStyle(0, { fillStyle: 'darkturquoise' });
 
   staveNotes[5].setStyle({ fillStyle: 'tomato', strokeStyle: 'tomato' });
-  beam3.setStyle({ shadowBlur: 20, shadowColor: 'blue' });
+  beam3.setStyle({ shadowBlur: 4, shadowColor: 'blue' });
```

Comment:
This is possibly going to break existing users that use `shadowBlur`. @ronyeh to capture this in the changelog.